react/components/Table/index.js

```diff
@@ -17,6 +17,7 @@ import { pluck } from 'ramda'
 
 const TABLE_HEADER_HEIGHT = 36
 const EMPTY_STATE_SIZE_IN_ROWS = 5
+const X_SCROLLBAR_OFFSET = 17
```

Comment:
the scrollbar size varies depending on the operating system (and user settings on top of that—some accessibility settings make the scrollbar large, for example).

can you find a way for this not to be hardcoded?

Reply:
yeah @lbebber i found some solutions, i think this is the cleanest one:
``` javascript
const getScrollbarWidth = () => window.innerWidth - document.documentElement.clientWidth;
```
what do you think?

i even tested here changing my scrollbar width on the browser :smile:

Reply:
@viniagostini LGTM but:
- check the existence of `window`, `document`, and `document.documentElement`, in case it runs on SSR
- check if it works across multiple browsers (I think it should, but just in case)
- make sure it always returns a number (and not `NaN` or something)

Reply:
Done! :smiley: